source/lesson-2-ruby-and-the-command-line.html.md.erb

```diff
@@ -337,15 +337,15 @@ again. It would be nice if we could give a particular task a name, and run it
 just by calling its name.
 
 In Ruby we do this with *methods*, and there are lots of them already built
-into the language. One example is `rand()`, which generates a random number:
+into the language. One example is `rand`, which generates a random number:
 
 ```
-irb(main):025:0> rand()
+irb(main):025:0> rand
 => 0.11487911496307956
 ```
 
 If we want to generate a random number less than a particular number, we can
-provide our maximum to `rand()` as an "argument":
+provide our maximum to `rand` as an "argument":
 
 ```
 irb(main):026:0> rand(100)
```

Comment:
Should the parentheses also be removed here for consistency?

Reply:
I think the clearer option would be to have parentheses where there are arguments passed - and no parentheses where there are no arguments passed.

Yes: `rand`, `rand(5)`
No: `rand()`, `rand 5`

Reply:
So:

Yes: `puts("hello world")`
No: `puts "hello world`

to be consistent? Or should we special case `puts`?